File: data/code_pairs/cpp/2040.cpp
#include <iostream>
#include <vector>
#include <algorithm>

int findMaxVal(const std::vector<int>& arr) {
    return *std::max_element(arr.begin(), arr.end());
}

int main() {
    std::vector<int> random_array = {100, 200, 300, 400, 500};
    int ans_val = findMaxVal(random_array);

    std::cout << "Maximum Value: " << ans_val << std::endl;

    return 0;
}

File: data/code_pairs/fortran/2040.f90
program main
  implicit none
  integer, parameter :: N = 5
  integer :: random_array(N), ans_val, i

  ! Predefined test array for simplicity
  random_array = [100, 200, 300, 400, 500]

  ans_val = 0
  do i=1, N
     ans_val = max(ans_val, random_array(i))
  enddo

  write(*,*) "Maximum Value: ", ans_val
end program main